src/components/v5/common/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import debounce from 'lodash.debounce';
+
+import FormError from '~v5/shared/FormError';
+
+import { TextareaProps } from './types';
+
+const displayName = 'v5.common.Textarea';
```

Comment:
Maybe create directory Fields in `v5/common` and put the Input, Select, Checkbox, and other fields component there?